apps/swirl-docs/src/lib/tokens/src/utils.ts

```diff
@@ -0,0 +1,23 @@
+const tokensLight = require("@getflip/swirl-tokens/dist/styles.light.json");
+
+export function getTokens(
+  tokensObject: any,
```

Comment:
**thought:** We could avoid this parameter if the function would create the object itself. Would make it easier to use the function.

Reply:
Thanks for pointing that out. i was able to simplify it even more :)